Shell/LineEditor.cpp

```diff
@@ -128,6 +130,7 @@ void LineEditor::tab_complete_first_token(const String& token)
     }
     for (int i = index + 1; i < m_path.size() && m_path[i].starts_with(token); ++i) {
         cut_mismatching_chars(completion, m_path[i], token.length());
+        m_suggestions.append(m_path[i]);
```

Comment:
Do we need this in the other loop too?

Reply:
This loops never seems to execute/do anything. Adding a `dbg()` print yields no results??